server/publications/spotlight.js

```diff
@@ -140,6 +140,11 @@ function searchUsers({ userId, rid, text, usernames }) {
 	}
 
 	const insiderExtraQuery = [];
+	const roomId = [rid];
+	const outsiderExtraQuery = [];
+	outsiderExtraQuery.push({
+		__rooms: { $nin: roomId },
+	});
```

Comment:
Since this is just used once, can you define `ousiderExtraquery` in place instead of defining the array and pushing into it?